### PR TITLE
[4.1] Template Override switches

### PR DESCRIPTION
**Pull request by brianteeman**
For some reason the switches were not displayed in the same way as other similar switches. This leads to confusion (reported elsewhere) and the need for creating additional css.

This Pr resolves that. Made against 4.1 as there are language string removals and additions and I dont know what the policy is for that in j4.

### Before
image

### After
image

cc @Scrabble96

### CODE REVIEW

File: administrator/components/com_templates/forms/source.xml [hunk 1]
```diff
@@ -39,21 +39,23 @@
 		<field
 			name="show_core"
 			type="radio"
+			label="COM_TEMPLATES_LAYOUTS_DIFFVIEW_CORE"
 			layout="joomla.form.field.radio.switcher"
 			default="0"
 			>
-			<option value="0">COM_TEMPLATES_LAYOUTS_DIFFVIEW_HIDE_CORE</option>
-			<option value="1">COM_TEMPLATES_LAYOUTS_DIFFVIEW_SHOW_CORE</option>
+			<option value="0">JHIDE</option>
```
Comment (sandewt) [inline administrator/components/com_templates/forms/source.xml:46]: filter="integer"

**Comment by Quy** [comment]
> PHP Notice:  Undefined property: Joomla\Component\Templates\Administrator\View\Template\HtmlView::$mediaFiles in \administrator\components\com_templates\tmpl\template\default.php on line 100

> Warning:  count(): Parameter must be an array or an object that implements Countable in \administrator\components\com_templates\tmpl\template\default.php on line 100

**Comment by brianteeman** [comment]
@quy I cannot replicate your error report and there is nothing in this PR remotely related to the line mentioned

**Comment by Quy** [comment]
Sorry false alarm. Applied to v4.0. OK in v4.1.

File: administrator/components/com_templates/forms/source.xml [hunk 2]
```diff
@@ -42,6 +42,7 @@
 			label="COM_TEMPLATES_LAYOUTS_DIFFVIEW_CORE"
 			layout="joomla.form.field.radio.switcher"
 			default="0"
+			filter="boolean"
```
Comment (sandewt) [inline administrator/components/com_templates/forms/source.xml]: Why a boolean ? Is this an appointment within Joomla ?

See [https://www.php.net/manual/en/language.types.boolean.php](https://www.php.net/manual/en/language.types.boolean.php)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b17288b43045bb310867c84eb529b3dd1e00de7a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36366">issues.joomla.org/tracker/joomla-cms/36366</a>.</sub>

**Comment by sandewt** [comment]
I want to test, but I can't find these switches. :sweat_smile:
How to do? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36366">issues.joomla.org/tracker/joomla-cms/36366</a>.</sub>

**Comment by brianteeman** [comment]
@sandewt They only display when you are working on a template override. So use the create override functionality and then select one of the newly created overrides and then you will see the switches in the top right

**Comment by sandewt** [comment]
With a little trick I can make the buttons visible and get the following. The text is not inline with the buttons. This depends on the screen size.

afbeelding

**Comment by Quy** [comment]
@sandewt Please `run npm i` to compile the CSS changes or download/install the prebuilt package.

**Comment by chmst** [comment]
I am not in favour of this kind of switches. Switches should be for yes/no only (I know that we have several on other places, and i think it is always wrong).

Why not use a label?  like "Hide original file: yes/no?"

**Comment by brianteeman** [comment]
> I know that we have several on other places, and i think it is always wrong).

so propose a change. this pr is consistent with current usage

**Comment by chmst** [comment]
What is current usage?

**Comment by brianteeman** [comment]
> What is current usage?

Verbs shoud be avoided in labels

**Comment by chmst** [comment]
Probably there is a reason for this rule, and maybe I am wrong with my my personal opinion, that switches should be used only for yes/no and lists for text-alternatives. 

If you were German I could help very easily, we can make a noun from every verb :)  "Versteckthaltung der Originaldatei"  

I will start a discussion for that, because I think it is important. But not directly related to this PR.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 9b37b14fe4d7233820432237c6aef4f7170eec7b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36366">issues.joomla.org/tracker/joomla-cms/36366</a>.</sub>

**Comment by chmst** [comment]
This solution is better than before.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36366">issues.joomla.org/tracker/joomla-cms/36366</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36366">issues.joomla.org/tracker/joomla-cms/36366</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thanks

**Pull request merged**